litrpc/netcmds.go

```diff
@@ -64,6 +65,25 @@ func (r *LitRPC) Connect(args ConnectArgs, reply *StatusReply) error {
 	return nil
 }
 
+// ------------------------- name a connection
+type AssignNicknameArgs struct {
+	Peer     uint32
+	Nickname string
+}
+
+func (r *LitRPC) AssignNickname(args AssignNicknameArgs, reply *StatusReply) error {
+	err := r.Node.SaveNicknameForPeerIdx(args.Nickname, args.Peer)
+	if err != nil {
+		return err
+	}
+
+	r.Node.RemoteCons[args.Peer].Nickname = args.Nickname
```

Comment:
I don't think so, in that case this error would be returned by the SaveNicknameForPeerIdx call above
https://github.com/mit-dci/lit/pull/47/files#diff-879af1248dab96ee2c9e0625c12f4bfeR281
and AssignNickname would already exit